tests/integration/test_case.go

```diff
@@ -338,6 +339,32 @@ type Request struct {
 	ExpectedError string
 }
 
+// GenerateDocs is an action that will trigger generation of documents.
+type GenerateDocs struct {
+	// NodeID may hold the ID (index) of a node to execute the generation on.
+	//
+	// If a value is not provided the docs generation will be executed against all nodes,
+	NodeID immutable.Option[int]
+
+	// Options to be passed to the auto doc generator.
+	Options []gen.Option
+
+	// The list of collection names to generate docs for.
+	// If not provided, docs will be generated for all collections.
+	ForCollections []string
+}
+
+// GeneratePredefinedDocs is an action that will trigger generation of documents.
```

Comment:
todo: It is not obvious to me what this action does, it says it generates documents, yet it takes a set of documents.  If you are providing it a set of documents, what does it generate, and how many of them?

Reply:
It doesn't multiply their number?  It just converts them from map to doc?

Can you please document this on the struct/action, I'd rather not have to read through the readme (which I may forget even exists) in order to understand this when reading tests.

Reply:
moved to another package to make is more distinctive